### DISCUSSION TITLE
help me understand this platform a little better - coming from a 'different platform'

### DISCUSSION THREAD

**Discussion by s-o-w** [iTwin/itwinjs-core#3485] [General] [answered]
HI all!  I sort of just stumbled onto/into this as I was searching for some information on digital twins.  I have a decent amount of experience with the Autodesk forge platform, and I'm trying to wrap my head around the differences here.  I see huge potential for this for our clients that are using Microstation/Bentley products and I think I see that I can create iModels that also incorporate Autodesk assets as well, which is good for us because we have to do both.  I am very new to this platform, so some of these questions are probably going to be elementary, but I want to ask anyway to perhaps help others coming behind me.

- Can I indeed create iModels that use Autodesk files?  If so, which ones are currently supported?
- It looks to me like, conceptually, an iModel holds ALL of the data, assets, drawings, geospatial, etc for a given "thing" , where thing in my case would be a power substation or even a section of a power grid that has 1 to N substations, distribution networks, and connection power lines.  Is that the case?
- what is the useful size limit, is there one?
- once I create an iModel and then create an application that uses it, do I have to have a bentley account to host that model/app for use?

thank you in advance for any information here, I think this is a great things and I'm really looking forward to trying some stuff out!

**Comment by kabentley** [comment, marked as answer]
> Can I indeed create iModels that use Autodesk files? If so, which ones are currently supported?

Yes, we have free connectors for AutoCAD (DWG) and Revit, and IFC. BTW, a single iModel can, and often does, contain data from many disparate source applications.

> It looks to me like, conceptually, an iModel holds ALL of the data, assets, drawings, geospatial, etc for a given "thing" , where thing in 
my case would be a power substation or even a section of a power grid that has 1 to N substations, distribution networks, and connection power lines. Is that the case?

Yes. Though generally we say that an iModel is appropriate for "engineered" assets. That is, data that has an "edit-review-approve" transaction model. Real-time (aka "OT") or enterprise (aka "IT") data belongs in other repositories, but the iModel can act as the "spatial backbone" for visualization, reporting and analysis.

> what is the useful size limit, is there one?

There is no hard limit, and most common operations generally use indexes. But some things can be size dependent. The biggest iModel I'm aware of is around 30Gb. Others may provide their real-world experiences.

> once I create an iModel and then create an application that uses it, do I have to have a bentley account to host that model/app for use?

No. As opposed to Forge, for example, an iModel can be a local file on your computer, and an iTwin.js application can be run on a desktop or mobile device with no external dependencies. However Bentley does have several services we offer for a fee, including hosting for web-visualization and iModelHub for managing user access and change tracking, among others.

HTH,
Keith

**Reply by s-o-w** [reply]
fantastic, thank you Keith!  I am going to try to work on some samples/proof of concept applications using some electrical designs and physical substation design models.  Most of our clients are doing electrical system modeling and 3D substation equipment modeling with autodesk products, but there are a few that are doing stuff with Microstation drawings as well.  If we can show a solution that can use either or or even both, I think we will be on to something.  Will report back with any findings, much obliged!

**Reply by tim-basic** [reply]
@kabentley Is it possible to convert autodesk files to an imodel file without using a desktop app or bentley hosted web services? Are there example repos for how to do this?
